portfolio/src/main/webapp/blog.html

```diff
@@ -19,7 +19,6 @@
         </ul>
     </nav>
     <header class="card px-5 py-5 my-5 text-center">
-       
```

Comment:
Same here, a commit that only removes an empty line. Did you miss something?

Reply:
Fixed it.